YML.md

```diff
@@ -30,28 +39,32 @@ As you can see, the `.yml` file consists of three sections:
 
-Optional. Either `binpatch: true` or `union: true`. These keys enable workarounds that make it possible to run applications from different, changing places in the filesystem (i.e., make them relocateable) that are not made for this. For example, some applications contain hardcoded paths to a compile-time `$PREFIX` such as `/usr`. This is generally discouraged, and application authors are asked to use paths relative to the main executable instead. Libraries like binreloc exist to make this easier. Since many applications are not relocateable yet, there are workarounds which can be used by one of these keys:
+* `binpatch: true`  indicates that binaries in the AppImage should be patched to replace the string `/usr` by the string `././`,  an `AppRun` file should be put inside the AppImage that does a `chdir()` to the `usr/` directory of inside AppDir before executing the payload application. The net effect is this that applications can find their resources in the  `usr/` directory inside the AppImage as long as they do not internally use `chdir()` operations themselves.
+* `union: true` indicates that an `AppRun` file should be put inside the AppImage that tries to create the impression of a union filesystem, effectively creating the impression to the payload application that the contents of the AppImage are overlayed over `/`. This can be achieved, e.g., using `LD_PRELOAD` and a library that redirects filesystem calls. This works as long as the payload application is a dynamically linked binary.
 
-* `binpatch: true`  indicates that binaries in the AppImage should be patched to replace the string `/usr` by the string `././`,  an `AppRun` file should be put inside the AppImage that does a `chdir()` to the `usr/` directory of inside AppDir before executing the payload application. The net effect is this that applications can find their resources in the  `usr/` directory inside the AppImage as long as they do not internally use `chdir()` operations themselves
-* `union: true` indicates that an `AppRun` file should be put inside the AppImage that tries to create the impression of a union filesystem, effectively creating the impression to the payload application that the contents of the AppImage are overlayed over `/`. This can be achieved, e.g., using `LD_PRELOAD` and a library that redirects filesystem calls. This works as long as the payload application is a dynamically linked binary 
 
 ### Ingredients section
 
-Describes how to acquire the binary ingredients that go into the AppImage. Binary ingredients can be archives like `.zip` files, packages like `.deb` files, debian repositories, and Ubuntu PPAs.
+Describes how to acquire the binary ingredients that go into the AppImage. Binary ingredients can be archives like `.zip` files, packages like `.deb` files or APT repositories like Debian package archives or PPAs.
```

Comment:
Is "Debian" the official spelling? I always find it irritating when a brand logo uses lowercase and copy text uses Uppercase for the same name...